cluster.tf

```diff
@@ -15,7 +15,7 @@ resource "aws_eks_cluster" "this" {
   tags                      = var.tags
 
   vpc_config {
-    security_group_ids      = compact([local.cluster_security_group_id])
+    security_group_ids      = compact(concat(tolist([local.cluster_security_group_id]), local.cluster_additional_security_group_ids))
```

Comment:
Maybe more clear to read and understand code will be making whole merging operation in `locals.tf` and here just pass one variable something like: `security_group_ids = local.cluster_security_group_ids` ?